cmd/lakectl/cmd/metastore.go

```diff
@@ -208,6 +248,17 @@ func init() {
 	_ = metastoreDiffCmd.Flags().String("to-schema", "", "destination schema name ")
 	_ = metastoreDiffCmd.Flags().String("to-table", "", "destination table name [default is from-table]")
 
+	metastoreCmd.AddCommand(metastoreCopyAllCmd)
+
+	_ = metastoreCopyAllCmd.Flags().String("from-address", "", "source metastore address")
+	_ = metastoreCopyAllCmd.MarkFlagRequired("from-address")
+	_ = metastoreCopyAllCmd.Flags().String("to-address", "", "destination metastore address")
+	_ = metastoreCopyAllCmd.MarkFlagRequired("to-address")
+	_ = metastoreCopyAllCmd.Flags().String("schema-filter", "*", "filter for schemas to copy in regex")
+	_ = metastoreCopyAllCmd.Flags().String("table-filter", "*", "filter for tables to copy in regex")
```

Comment:
If filter syntax is regexp - '*' as default value is not a good one.
We can go with an empty value which means no filter or '.*'.

Reply:
It is filtered using hive metastore logic, which is somewhere between wildcard and regex. Passing an empty filter will return no results.